io.catenax.batch/2.0.0/Batch.ttl

```diff
@@ -0,0 +1,180 @@
+    samm:description "Timestamp of the manufacturing date as the final step in production process (e.g. final quality check, ready-for-shipment event)"@en;
+    samm:characteristic samm-c:Timestamp;
+    samm:exampleValue "2022-02-04T14:48:54"^^xsd:dateTime.
+:country a samm:Property;
+    samm:preferredName "Country code"@en;
+    samm:description "Country code where the part was manufactured"@en;
+    samm:characteristic :ProductionCountryCodeTrait;
+    samm:exampleValue "HUR".
+:manufacturerPartId a samm:Property;
+    samm:preferredName "Manufacturer Part ID"@en;
+    samm:description "Part ID as assigned by the manufacturer of the part. The Part ID identifies the part (as designed) in the manufacturer`s dataspace. The Part ID does not reference a specific instance of a part and thus should not be confused with the serial number or batch number."@en;
+    samm:characteristic :PartIdCharacteristic;
+    samm:exampleValue "123-0.740-3434-A".
+:nameAtManufacturer a samm:Property;
+    samm:preferredName "Name at Manufacturer"@en;
+    samm:description "Name of the part as assigned by the manufacturer"@en;
+    samm:characteristic :PartNameCharacteristic;
+    samm:exampleValue "Mirror left".
+:classification a samm:Property;
+    samm:preferredName "Classifcation"@en;
+    samm:description "The classification of the part type according to STEP standard definition"@en;
+    samm:characteristic :ClassificationCharacteristic;
+    samm:exampleValue "raw material".
+:KeyTrait a samm-c:Trait;
+    samm:preferredName "Key Trait "@en;
+    samm:description "Trait that ensures the usage of predefined keys. "@en;
+    samm-c:baseCharacteristic :KeyCharacteristic;
+    samm-c:constraint ext-batch2:KeyRegularExpression.
+:ValueCharacteristic a samm:Characteristic;
+    samm:preferredName "Value Characteristic"@en;
+    samm:description "The value of an identifier."@en;
+    samm:dataType xsd:string.
+:ProductionCountryCodeTrait a samm-c:Trait;
+    samm:preferredName "Production Country Code Trait"@en;
+    samm:description "Trait to ensure standard data format for country code"@en;
+    samm-c:baseCharacteristic :CountryCodeCharacteristic;
+    samm-c:constraint :CountryCodeRegularExpression.
+:PartIdCharacteristic a samm:Characteristic;
+    samm:preferredName "Part ID Characteristic"@en;
+    samm:description "The part ID is a multi-character string, ususally assigned by an ERP system"@en;
+    samm:dataType xsd:string.
+:PartNameCharacteristic a samm:Characteristic;
+    samm:preferredName "Part Name Characteristic"@en;
+    samm:description "Part Name in string format from the respective system in the value chain"@en;
+    samm:dataType xsd:string.
+:ClassificationCharacteristic a samm-c:Enumeration;
+    samm:preferredName "Classification Characteristic"@en;
+    samm:description "A part type must be placed into one of the following classes: 'component', 'product', 'software', ?assembly?, 'tool', or 'raw material'."@en;
```

Comment:
?assembly? -> question marks? still under discussion or what does that mean?

Reply:
done